### ISSUE TITLE
Adam setting:beta1=0 not work

### ISSUE THREAD

**Issue by Athenanna** [NVlabs/stylegan2-ada-pytorch#106]
Hi, thanks for your great work!  I'm confused with the setting of Adam, why stylegan family use beta1=0( Adam(beta1=0.**net_rate, beta2=0.99**net_rate)) ?   Why not 0.9 as default, is there any particular reason?

**Model:** 
I trained an image2image model based on stylegan2 (input image to the encoder which could be unet,  the encoder outputs latent code for stylegan2-generator). Data: gt: FFHQ, input: FFHQ degraded. 1* loss: adv loss & 0.1* l1 loss

**Problems:**
all loss get NaN, seems related to the setting of Adam: 
I found that the losses reduced first and suddenly NAN at 1k step, and the output image turned to be black (at first, got normal output image, then  black).
Then I set the Adam(beta1=0.9**net_rate, beta2=0.99**net_rate), the loss remains reducing by now (6.5k step), and the output image looks good. 
I pretrained the stylegan2 from  scratch  with beta1=0 on FFHQ, which is going well,  but beta1=0 seems not suitable for transfer learning（the image2image model）。
could you please help if you have some research and conclusions on Adam settings?
Thanks!

**env info**
 - Linux 
 - PyTorch 1.7.1, python 3.6
 **- tensorflow official repos for stylegan2**  https://github.com/NVlabs/stylegan2
 -  CUDA 11.0
